src/components/application/application.js

```diff
@@ -1,9 +1,25 @@
 import './application.scss';
 
-function Application() {
+function Application({ logout, user = {} }) {
   return (
     <div className="application">
-      Login Challenge
+      <header className="header">
+        <h1>Application</h1>
+        <div className="user">
+          <p className="userLink">{user.email}</p>
+          <button
+            type="button"
+            onClick={() => {
+              logout();
+            }}
+          >
+            Logout
+          </button>
+        </div>
+      </header>
+      <main className="main">
```

Comment:
surely we can't need a class `main` for the `main` element?

Reply:
Possibly not, more than one main element would be odd.

I'm used to working with CSS Modules, avoiding using element based rules and not worrying about class name conflicts. I had planned on adding CSS module support, just hadn't got around to it.

I've removed the main class for now.